src/frontend/src/menubar/index.tsx

```diff
@@ -0,0 +1,14 @@
+import AppIcon from "./AppIcon";
+import DarkLightToggle from "./DarkLightToggle";
+
+
+function SpaceSeparator() {
+    return <div className="grow" />;
+}
+
+
+export const MenuComponents = [
```

Comment:
Ich halte es für nicht so optimal, dass es separat exportiert wird, um in App.tsx zusammengefügt zu werden. Können wir das anders gliedern?
Ich würde die Folgende Ordnerstruktur vorschlagen:
/menubar/
 |__ component/
> AppIcon.tsx
> DarkLightToggle.tsx

 |__ Menubar.tsx
 |__ index.ts



Dann kannst du den SpaceSeperator und andere Formatierungen in Menubar.tsx übernehmen.

Die icons würde meiner Meinung nach auch im frontend/ ordner neben src/ Sinn machen